charts/document-engine/CHANGELOG.md

```diff
@@ -86,6 +88,12 @@
   - [2.0.0](#200)
     - [Changed](#changed-23)
 
+## 3.4.0 (2025-02-11)
+
+### Changed
+
+* [Document Engine 1.6.0](https://www.nutrient.io/changelog/document-engine/#1.6.0)
```

Comment:
Is redirect in place there?

Reply:
977911149c0a344ebba2a71f9ffd54276d986638